app/helpers/project_helper.rb

```diff
@@ -77,4 +77,18 @@ def confirmed_date_original_date(project)
     original_date = project.provisional_date
     "#{confirmed_date.to_fs(:govuk_short_month)} (#{original_date.to_fs(:govuk_short_month)})"
   end
+
+  def directive_academy_order_responses
```

Comment:
As these are just yes & no I'm wondering if it'd be easier/less code to drop them and use `yes_no_responses` again?

Reply:
It's confusing for sure! But the actual responses are 'directive academy order' and 'academy order', we have to keep mentally mapping between the two because the attribute is `directive_academy_order` and boolean, but we must have decided this other way is better and not felt it worth changing the database to something like an enum.